.github/workflows/ci.yml

```diff
@@ -43,6 +43,8 @@ jobs:
 
       - name: Setup Rust toolchain and cache
         uses: actions-rust-lang/setup-rust-toolchain@v1.5.0
+        with:
+          rustflags: ""
```

Comment:
Used double quotes to keep consistency. (Double quotes are mainly used inside this file)